### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by kelibst**
First, pull request
- Private events project
- has model validation and authentication test

### CODE REVIEW

**Comment by pelzolga123** [comment]
## Hi Keli, 
### The project looks very nice! Good job :+1: 
Still, there are a few more changes required:
* [ ] The user's show page should display events the user has created and events that the user is attending to separated in past and upcoming ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
* [ ] Add instructions on how to run tests to Readme file and make sure to remove duplicate `private-events` header.
* [ ] One error occurred while running test, it should be fixed:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  end
+
+  def new
+    @user = current_user
+    @event = Event.new
+  end
+
+  def create
+    @user = current_user
+    @event = @user.events.new(event_params)
```
Comment (pelzolga123) [inline app/controllers/events_controller.rb:17]: Use #build method on association, i.e. `current_user.events.build(event_params)` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/build_method_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        format.html { redirect_to user_path(@user), notice: "#{@user.username} has successfully signed up" }
+      else
+        format.html { render :new }
+      end
+    end
+  end
+
+  def show
+    @events_created_by_user = @user.events.order('created_at DESC')
+  end
```
Comment (pelzolga123) [inline app/controllers/users_controller.rb:22]: * Should list events previoulsy attendented by user **using event model scope**, i.e. `@past_events = current_user.attended_events.previous` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
* Should list upcoming events previously attended by user **using event model scope**, i.e. `@upcoming_events = current_user.attended_events.upcoming` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

Comment (kelibst) [reply app/controllers/users_controller.rb:22]: This is honestly the most helpful review I have had in a long time.
